app/javascript/src/components/Common/Menubar.jsx

```diff
@@ -0,0 +1,75 @@
+// @ts-nocheck
+import React, { useState } from "react";
+
+import { Search, Settings, Plus } from "neetoicons";
+import { Typography } from "neetoui";
+import { MenuBar } from "neetoui/layouts";
+
+function Menubar({ title, setCategory, options }) {
+  const [isSearchCollapsed, setIsSearchCollapsed] = useState(true);
+
+  return (
+    <MenuBar showMenu={true} title={title}>
+      {options?.heads?.map((head, idx) => (
+        <MenuBar.Block
+          label={head.title}
+          key={idx}
```

Comment:
Index should be last resort:
https://medium.com/wesionary-team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9